.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -24,6 +27,9 @@ jobs:
   upload-release-vulnerabilities:
     needs: read-release-versions
     runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.61
+      options: -v /tmp:/tmp # make sure Claicore blob can access tmp folder
```

Comment:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontainervolumes